Dockerfile

```diff
@@ -1,4 +1,10 @@
+# This dockerfile is meant to be used in publish-docker.yml workflow which reuses artifacts created by release.yml
+# This way we can avoid rebuilding the CLI specifically for docker
+
 FROM alpine:3.20
 
+# binary should be retrieved by download-artifact action by this point
 COPY ./railway /usr/bin/railway
-RUN test -f /usr/bin/railway
+
+# ensure it has exec permission
+RUN chmod +x /usr/bin/railway
```

Comment:
To save a line, use the chmod flag in the COPY instruction.

Reply:
done